flake.nix

```diff
@@ -11,7 +11,7 @@
     flake-utils = { url = "github:numtide/flake-utils"; };
     hydra.url = "hydra";
     hackage = {
-      url = "github:input-output-hk/hackage.nix";
+      url = "github:input-output-hk/hackage.nix/hkm/nixify";
```

Comment:
I guess here we use `hkm/nixify` branch to test this PR while https://github.com/input-output-hk/nix-tools/pull/118 isn't merge into `master`, but PR branch name is `hkm/nixify-hackage-to-nix`

Reply:
I made that branch manually on hackage.nix (by using the branch of nix-tools). Once hydra is happy with this PR I'll put this ref back to master and run bors try again to make sure the change here still works with old hackage.nix (some people like to pin that to older versions even when updating haskell.nix).

Once `nix-tools` PR is merged I'll update this PR to include that.

Then we can merge this PR and on run the daily update.  It should make a commit like my manual one on the hackage.nix/hkm/nixify branch.